src/Uno.UI/UI/Xaml/Controls/TextBox/SinglelineTextBoxView.iOS.cs

```diff
@@ -51,6 +51,22 @@ public override string Text
 			}
 		}
 
+		public override bool ResignFirstResponder()
+		{
+			if (AttributedText?.MutableCopy() is NSMutableAttributedString newAttributedText)

```

Comment:
Can you please you add a comment on what is the purpose of the code block (or move it to a method with a descriptive name)?

Reply:
@MartinZikmund Is `// https://stackoverflow.com/a/61762392/5108631` a good enough comment? 😄

Reply:
You should be able to add a Runtime Test for that, using big squares and pixel validations, using the new [RenderTargetBitmap feature tests](https://github.com/unoplatform/uno/blob/aa9311da1926bd52df96ec152cb7585d3c8d74a3/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_Opacity.cs#L37) (iOS/Android/Skia only, wasm is not yet supported)

As for the documentation, it's probably best to extract that code into a separate method that states the intent (and call it from `ResignFirstResponder`).